### PR TITLE
Created rohith.json

### PR DESCRIPTION
**Pull request by rohithp7777**

### CODE REVIEW

**Review by shajanjp — CHANGES_REQUESTED** [review]
Is https://rohithp7777.github.io/ working?
Refer https://pages.github.com/ to make one.

**Comment by rohithp7777** [comment]
> Is https://rohithp7777.github.io/ working?
> Refer https://pages.github.com/ to make one.

is firebase okay

> Is https://rohithp7777.github.io/ working?
> Refer https://pages.github.com/ to make one.

sorry for the trouble firebase is okay ig ?

**Comment by rohithp7777** [comment]
> Is https://rohithp7777.github.io/ working?
> Refer https://pages.github.com/ to make one.

Oh sorry, is firebase hosted one fine?

File: domains/rohith.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+{
+    "description": "Rohith's personal website",
+    "repo": "https://github.com/rohithp7777",
+    "owner": {
+        "username": "rohithp7777",
+        "email": "rohithp8899@gmail.com"
+    },
+    "record": {
+        "CNAME": "https://rohithp7777.github.io",
+        "URL": "https://rohith-1c974.web.app/"
```
Comment (phenax) [inline domains/rohith.json:11]: Please make sure firebase supports custom domains. If not, you can use github pages.

```suggestion
        "CNAME": "rohith-1c974.web.app"
```

**Review by phenax — CHANGES_REQUESTED** [review]

**Review by phenax — APPROVED** [review]

**Pull request merged**